geom/accessor_test.go

```diff
@@ -21,12 +21,12 @@ func TestLineAccessor(t *testing.T) {
 	line := geomFromWKT(t, "LINESTRING(1 2,3 4)").AsLine()
 	t.Run("start", func(t *testing.T) {
 		got := line.StartPoint()
-		want := Coordinates{XY{1, 2}}
+		want := Coordinates{XY: XY{1, 2}, Type: DimXY}
```

Comment:
Nit: The repeated use of `Coordinates{XY: XY{x, y}, Type: DimXY}` feels cumbersome, is there any way to reduce it?

Reply:
Good point, I've now created a helper, which removes some of the clutter.